debezium-core/src/main/java/io/debezium/transforms/strategy/DeleteTombstoneExtractRecordStrategy.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.transforms.strategy;
+
+import org.apache.kafka.connect.connector.ConnectRecord;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.debezium.DebeziumException;
+import io.debezium.transforms.ExtractNewRecordStateConfigDefinition.DeleteTombstoneHandling;
+
+/**
+ * A default implementation of {@link AbstractExtractRecordStrategy}
+ *
+ * @author Harvey Yue
+ */
+public class DeleteTombstoneExtractRecordStrategy<R extends ConnectRecord<R>> extends AbstractExtractRecordStrategy<R> {
```

Comment:
Isn't this more a type of `DeleteHandlingStrategy` not a entire `ExtractRecordStrategy`? E.g. `DropTombstoneDeleteHandlingStrategy` here?